### ISSUE TITLE
ZREP_SEND_FLAGS fix example

### ISSUE THREAD

**Issue by kocio-pl** [bolthole/zrep#176]
Hi,

Thanks for this tool! :tada: .

It looks to me that example value of `ZREP_SEND_FLAGS` does not work:
https://github.com/bolthole/zrep/blob/master/zrep#L100

Instead of:

`#ZREP_SEND_FLAGS=-c -raw`

it should be probably (adding quote marks and one more dash for `-raw`):

`#ZREP_SEND_FLAGS="-c --raw"`

**Comment by ppbrown** [comment]
Hello,

you didnt specify what os  and version of zfs you are running it on.
it is quite likely that some take -raw and others take --raw.

**Comment by kocio-pl** [comment]
Sorry for not including any debug details. :smile:  I use PVE 7.1, which is basically current Debian with some Proxmox additions and OpenZFS 2.1. Both default shell (bash) and ksh give me some errors:

```
ZREP_R=-R ZREP_SEND_FLAGS=-raw -c zrep init rpool/data host rpool/data
bash: -c: command not found
```
```
ksh ZREP_R=-R ZREP_SEND_FLAGS=-raw -c zrep init rpool/data host rpool/data
ksh: ZREP_SEND_FLAGS=-raw: not found
```
Changing the value does not help in ksh:
```
ksh ZREP_R=-R ZREP_SEND_FLAGS="--raw -c" zrep init rpool/data host rpool/data
ksh: ZREP_SEND_FLAGS=--raw -c: not found
```
but works for default shell:
```
ZREP_R=-R ZREP_SEND_FLAGS="--raw -c" zrep init rpool/data host rpool/data
Setting zrep properties on rpool/data
```
In the comment line before `--raw` is mentioned, so it's at least inconsistent with the example value:

> --raw is sometime used for encrypted filesystems

https://github.com/bolthole/zrep/blob/master/zrep#L99

**Comment by ppbrown** [comment]
you are not invoking it correctly, from a shell perspective.


you should use

ZREP_R=-R ZREP_SEND_FLAGS="-raw -c" zrep init rpool/data host rpool/data

or, make a wrapper, or put in your environment somehow,

export ZREP_R=-R
export ZREP_SEND_FLAGS="--raw -c"
#then
zrep init rpool/data host rpool/data

**Comment by kocio-pl** [comment]
Yes, that's exactly what I did after fixing (wrapper elegance aside):

> but works for default shell:
> 
> ZREP_R=-R ZREP_SEND_FLAGS="--raw -c" zrep init rpool/data host rpool/data
> Setting zrep properties on rpool/data

But it didn't work without quotes and this is what the example value shows. Uncommenting this value directly in the zrep also gave me error.

**Comment by ppbrown** [comment]
Ohhh.. I see.
thank you for pointing that out that I should have the commented out
version use quotes :)

**Comment by ppbrown** [comment]
fixed in ef5f4c7
